packages/venia-concept/src/ServiceWorker/Utilities/__tests__/htmlHandler.spec.js

```diff
@@ -0,0 +1,141 @@
+jest.mock('../messageHandler');
+
+import { HTML_UPDATE_AVAILABLE } from '@magento/venia-ui/lib/constants/swMessageTypes';
+
+import { cacheHTMLPlugin } from '../htmlHandler';
+import { sendMessageToWindow } from '../messageHandler';
+
+const matchFn = jest.fn();
+
+beforeAll(() => {
+    global.caches = {
+        match: matchFn
+    };
+    global.Request = function(url, { method, headers } = {}) {
+        this.url = url;
+        this.method = method;
+        this.headers = headers;
+    };
+    global.Response = function() {};
+});
+
+beforeEach(() => {
+    sendMessageToWindow.mockClear();
```

Comment:
Is `sendMessageToWindow` a mocked function here? Where do we mock it?

Reply:
In the first line, I mocked the whole `messageHandler.js` module.

Reply:
On the other hand, we don't need that line anymore. Removing it.